websites/docs-smoke-test/gatsby-config.js

```diff
@@ -1,5 +1,11 @@
 const isProd = process.env.NODE_ENV === 'production';
 const shouldEnableSearch = process.env.ENABLE_SEARCH === 'true';
+
+if (process.env.NODE_ENV === 'development') {
+  require('dotenv').config({
+    path: `.env`,
+  });
+}
```

Comment:
You should be able to use `.env.development` and Gatsby will load these env variables automatically. There should be no need to manually load them.

Can you double check? https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/

Reply:
I was actually reading this doc too and it says [here](https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#:~:text=For%20example%2C%20when%20configuring%20a%20plugin%20in%20gatsby%2Dconfig.js%3A) that this is an example of how to provide keys to plugins which is exactly our use case and they had to manually load them ... this is why I did so

it is still failing in the [build](https://github.com/commercetools/commercetools-docs-kit/runs/8151852021?check_suite_focus=true#step:9:231) so this might actually mean that its not able to load the production variables too so it might be even worth to try their code snippet
`require("dotenv").config({
  path: `.env.${process.env.NODE_ENV}`,
})`

CC: @timonrey